lib/imagery/sig.c

```diff
@@ -81,7 +82,7 @@ int I_free_signatures(struct Signature *S)
     return 0;
 }
 
-int I_read_one_signature(FILE * fd, struct Signature *S)
+int I_read_one_signature(FILE * fd, int have_oclass, struct Signature *S)
```

Comment:
Unlike the previous version which was relatively clear, this new version really needs Doxygen doc.